plugins/amazonq/codewhisperer/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererTelemetryTest.kt

```diff
@@ -531,7 +531,8 @@ class CodeWhispererTelemetryTest : CodeWhispererTestBase() {
 
         CodeWhispererCodeCoverageTracker.getInstance(project, CodeWhispererPython.INSTANCE).dispose()
 
-        val acceptedTokensSize = pythonResponse.completions()[0].content().length - deletedTokenByUser
+        val rawAcceptedTokenSize = pythonResponse.completions()[0].content().length
```

Comment:
null safe:
```suggestion
        val rawAcceptedTokenSize = pythonResponse.completions().firstOrNull()?.content()?.length
```
or if definitely has a completion
```suggestion
        val rawAcceptedTokenSize = pythonResponse.completions().first().content().length
```